x-pack/plugin/profiling/src/test/java/org/elasticsearch/xpack/profiling/GetStackTracesRequestTests.java

```diff
@@ -89,6 +92,82 @@ public void testParseXContentUnrecognizedField() throws IOException {
         }
     }
 
+    /*
```

Comment:
If the purpose is to ignore these, then I believe it should be possible to annotate them with `@Ignore` instead.